src/utils.js

```diff
@@ -0,0 +1,90 @@
+const fs = require("fs");
+const child_process = require("child_process");
+const path = require("path");
+const glob = require("glob");
+const semver = require("semver");
+
+function getTags() {
+    return child_process.execSync(`git tag`, {encoding: "utf8"}).split("\n").filter(Boolean);
+}
+
+function getWorkspaces() {
+    const pkgJson = JSON.parse(fs.readFileSync("package.json", "utf8"));
+    return pkgJson.workspaces.reduce((result, wsGlob) => {
+        return result.concat(glob.sync(wsGlob));
+    }, []);
+}
+
+// We assume that if a there's a tag for the current version of a package that
+// it's been published already.  In the future we should check if that version
+// exists on npm and attempt to republish the package if it isn't.
```

Comment:
Capture that in an issue.

Reply:
Captured in https://github.com/Khan/antwerp/issues/3.

Reply:
I can update the comment to link back to the GitHub issue.